aquacomputer_d5next.c

```diff
@@ -211,6 +218,29 @@ static const char *const label_fan_flow_speed_quadro[] = {
 	"Flow speed [l/h]"
 };
 
+struct fan_structure_offsets {
```

Comment:
`aqc_fan_structure_offsets`. It must be clear from which driver the struct is coming from.